paasta_tools/cli/cmds/push_to_registry.py

```diff
@@ -197,4 +243,26 @@ def is_docker_image_already_in_registry(service, soa_dir, sha):
             return True
         elif r.status_code == 404:
             return False  # No Such Repository Error
+
         r.raise_for_status()
+
+
+def is_docker_image_already_in_registries(service, soa_dir, sha):
```

Comment:
can you simplify this and put it in terms of ```where_does_docker_image_exist_and_does_not``` ?

Reply:
Fixed :) Please review.